Configuration/TypoScript/Toolbox/setup.txt

```diff
@@ -1,5 +1,7 @@
 plugin.tx_dlf_annotationtool.templateFile = EXT:dlf/Resources/Private/Templates/AnnotationTool.tmpl
+plugin.tx_dlf_fulltexttool.launchState = off
```

Comment:
Sorry, I cannot reproduce what this setting does:

* Without launchState: nothing happens. Ok.
* with launchState=off: the fulltext is shown initially and on every click on the button it is marked once more. So the color gets darker and darker (in the screenshot its getting more and more red)
* with launchState=on: the old behaviour shows up

So what is launchState=off good for?
![fulltext-launchstate](https://user-images.githubusercontent.com/3354500/102941310-b2d36d00-44b2-11eb-96d2-25a265a4f321.png)